src/styles/variables.scss

```diff
@@ -0,0 +1,113 @@
+@import '~vuetify/src/styles/styles.sass';
```

Comment:
```
@import '~vuetify/src/styles/settings/_colors.scss';
```

Reply:
Thanks @KaelWD. That will make it better.

Reply:
But, unfortunately, `$material-light` is not defined in `_colors.scss`. I'll search for it and import others too.


```
SassError: Undefined variable.
   ╷
40 │                 'solo-inverted-focused-text': map-get(map-get($material-light, 'text'), 'primary')

```

Reply:
I found it and the complete imports are as follows. Thanks!

```scss
@import '~vuetify/src/styles/settings/_colors.scss';
@import '~vuetify/src/styles/settings/_light.scss';
```